src/main/java/de/tum/in/www1/artemis/service/TutorLeaderboardService.java

```diff
@@ -112,8 +145,21 @@ else if (complaintResponsesView.getComplaintResponses() != null) {
                 }
             }
 
-            tutorLeaderBoardEntries
-                    .add(new TutorLeaderboardDTO(tutor.getId(), tutor.getName(), numberOfAssessments, numberOfAcceptedComplaints, numberOfComplaintResponses, points));
+            for (TutorLeaderboardAnsweredMoreFeedbackRequestsView moreFeedbackRequestsView : tutorLeaderboardAnsweredMoreFeedbackRequests) {
+                if (moreFeedbackRequestsView.getUserId().equals(tutor.getId())) {
+                    numberOfAnsweredMoreFeedbackRequests += moreFeedbackRequestsView.getAnsweredRequests();
+                    // answered complaints count only 1x
```

Comment:
I guess this is not a good idea. Answered more feedback requests should not give any points. Otherwise, we would reward tutors who have first provided bad feedback and then improve their feedback afterwards.

So please stick to the following rule:
- not answered more feedback requests give -1x points
- answered more feedback requests give 0 points

This basically means the tutor can "repair" the negative points by answering the request.

Reply:
Thank you for the explanation. I also added a short comment in the code